sync/odoo_syncer.go

```diff
@@ -0,0 +1,8 @@
+package sync
+
+import "github.com/vshn/appuio-odoo-adapter/odoo/model"
+
+// OdooSyncer synchronizes various reporting facts with Odoo API.
+type OdooSyncer struct {
+	odoo *model.Odoo
+}
```

Comment:
Maybe a better name would be `InvoiceCategoryReconciler` as it basically does reconcile those categories in Odoo, with Reporting DB as authoritative data source.